ionic2/app/pages/search/search.page.ts

```diff
@@ -64,7 +64,7 @@ export class SearchPage {
 
   selectPokemon(pokemon:Pokemon) {
     this.cancelRequests();
-    this.navCtrl.setRoot(WikiPage, {pokemonId: pokemon.pokemonId});
+    this.navCtrl.setRoot(PokeDetailPage, {pokemonId: pokemon.pokemonId});
```

Comment:
It would be nice if you could pop both the PokeDetailPage and the WikiIndexPage on the navigation stack. This way we would have a back button on the PokeDetailPage that takes the user back to the WikiIndexPage.